app/lib/actions/katello/host/erratum/install.rb

```diff
@@ -9,10 +9,19 @@ def plan(host, errata_ids)
             Type! host, ::Host::Managed
 
             action_subject(host, :errata => errata_ids)
-            plan_action(Pulp::Consumer::ContentInstall,
-                        consumer_uuid: host.content_facet.uuid,
-                        type:          'erratum',
-                        args:          errata_ids)
+            if Setting['erratum_install_batch_size'] && Setting['erratum_install_batch_size'] > 0
```

Comment:
@anandkumaragrawal The difference is seen when `Setting['erratum_install_batch_size']` is nil. Without the first check, you would get an `undefined method '>' for nil:NilClass` error.